### ISSUE TITLE
Not displaying files.

### ISSUE THREAD

**Issue by Jacklack3** [lempamo/NeoTool#7]
I have a bug with Neotool, it can tell if my password is wrong, however when I sign in it doesn't show my files. Doing refresh files makes it say Neocities has an error on their site but then shows this error.
```System.NullReferenceException: Object reference not set to an instance of an object.
   at NeoTool.MainForm.PopulateTreeView()
   at NeoTool.MainForm.kryptonButton6_Click(Object sender, EventArgs e)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)```

**Comment by lempamo** [comment]
Looks like the problem is the app is trying to use info that it can't get (due to invalid auth, offline server, etc.). When I have the time I'll throw some Exceptions and try{}catch{} statements in, that should fix it.

**Comment by lempamo** [comment]
fixed as of 1.1.1
